newrelic/resource_newrelic_synthetics_alert_condition.go

```diff
@@ -0,0 +1,168 @@
+package newrelic
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	newrelic "github.com/paultyng/go-newrelic/api"
+)
+
+func resourceNewRelicSyntheticsAlertCondition() *schema.Resource {
```

Comment:
Please add a test for the new resource. See https://github.com/terraform-providers/terraform-provider-newrelic/blob/master/newrelic/resource_newrelic_nrql_alert_condition_test.go for a similar example. Thanks!